test/reporters/list.spec.js

```diff
@@ -138,10 +138,16 @@ describe('List reporter', function() {
       var actual = {a: 'actual'};
       var expected = {a: 'expected'};
       var checked = false;
+      var stringifyDiffs = 'stringifyDiffObjs';
```

Comment:
As this is used nowhere else, migrate it inside the event handler callback.

Reply:
But then the same value is declared and assigned "every single time" a callback runs (and with this test it happens quite a lot). Isn't it better its assigned beforehand and then re-used?